### PR TITLE
Initial Sydney 2016 Content

### PR DESCRIPTION
**Pull request by glasnt**
🎉 🐨

### CODE REVIEW

File: data/events/2016-sydney.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#  - name: program
+  - name: propose
+  - name: location
+#  - name: registration
+  - name: sponsor
+  - name: contact
+  - name: conduct
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (bridgetkromhout) [inline data/events/2016-sydney.yml]: We ask that cities list everyone on the team to correspond with the members of the organizer email list. Can you add the other folks please, including yourself? :)

**Comment by glasnt** [comment]
@bridgetkromhout Updated (and rebased 🙃)

**Comment by bridgetkromhout** [comment]
Good deal.

**Comment by bridgetkromhout** [comment]
Oh, you'll want to correct the location latitude/longitude in the data yaml. The default places it on the map at Chicago.

**Comment by bridgetkromhout** [comment]
And maybe remove the "Even if you don’t propose, please consider" text on the talk proposal page.

**Pull request merged**